src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -0,0 +1,322 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import * as keyboardKey from 'keyboard-key'
+import * as PropTypes from 'prop-types'
+
+import eventStack from '../../eventStack'
+
+import {
+  getNextElement,
+  getFirstTabbable,
+  getLastTabbable,
+  getWindow,
+  focusAsync,
+  HIDDEN_FROM_ACC_TREE,
+} from './focusUtilities'
+
+import { IFocusTrapZone, IFocusTrapZoneProps } from './FocusTrapZone.types'
+import getUnhandledProps from '../../getUnhandledProps'
+import getElementType from '../../getElementType'
+import * as customPropTypes from '../../customPropTypes'
+
+export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}>
+  implements IFocusTrapZone {
+  private static _focusStack: FocusTrapZone[] = []
+  private _root: { current: HTMLElement | null } = { current: null }
+  private _previouslyFocusedElementOutsideTrapZone: HTMLElement
+  private _previouslyFocusedElementInTrapZone?: HTMLElement
+  private windowElement: Window | null
+
+  private createRef = elem => (this._root.current = ReactDOM.findDOMNode(elem) as HTMLElement)
```

Comment:
is it intentional that you would like to return dom node from this call? According to the place where method is used, its not. In that case we should consider to use the following form instead:

```js
private createRef = elem => { this._root.current = ReactDOM.findDOMNode(elem) as HTMLElement }
```

Reply:
right, good catch